lib/screens/home/pages/relation/relation_page.dart

```diff
@@ -163,8 +165,8 @@ class _RelationPageState extends State<RelationPage> {
                     GestureDetector(
                       onTap: () {
                         if (!task.isDone) {
-                          context.toast("hey");
```

Comment:
@Techno-Disaster I think taking off this line was supposed to be for another issue which we discussed in last meeting right?

Reply:
@HaripriyaB yes, but IMHO it's just 1 small line and it doesn't really make any sense to have a whole new Issue for it.

Reply:
yes that makes sense...